internal/build/image_builder.go

```diff
@@ -215,10 +217,17 @@ func (d *dockerImageBuilder) addRemainingSteps(df dockerfile.Dockerfile, remaini
 }
 
 // Tag the digest with the given name and wm-tilt tag.
-func (d *dockerImageBuilder) TagImage(ctx context.Context, ref reference.Named, dig digest.Digest) (reference.NamedTagged, error) {
-	tag, err := digestAsTag(dig)
-	if err != nil {
-		return nil, errors.Wrap(err, "TagImage")
+func (d *dockerImageBuilder) TagImage(ctx context.Context, ref reference.Named, dig digest.Digest, tagLatest bool) (reference.NamedTagged, error) {
```

Comment:
maybe there should be two separate methods?
TagImage(reference.Named, tag sdig Digest)
TagImageFromDigest(reference.Named, dig Digest)

presumably, the second one could use the first one

Reply:
no longer relevant after refactor